Scripts/interactable.gd

```diff
@@ -0,0 +1,10 @@
+class_name interactable
```

Comment:
Can you capitalize `Interactable` like so? Class names should be Pascal Case.

Pascal Case Examples:
- `TestThing`
- `Monkey`
- `SnowballFight`
- ThisIsAVeryStupidNameForAnythingInACodingProjectButThisDoesDemonstrateWhatPascalCaseIsAndWhyItsUseful`

The only reason I case here is that `Interactable` is how other scripts with reference this one.